apps/studio/components/interfaces/Integrations/CronJobs/CronJobScheduleSection.tsx

```diff
@@ -1,107 +1,221 @@
+                          <Button
+                            type="outline"
+                            onClick={() => {
+                              setPresetValue(preset.expression)
+                              setInputValue(preset.expression)
+                              setUseNaturalLanguage(false)
+                            }}
+                          >
+                            {preset.name}
+                          </Button>
+                        </li>
+                      ))}
+                    </ul>
+                  </div>
+                  <Accordion_Shadcn_ type="single" collapsible className="mt-2">
+                    <AccordionItem_Shadcn_ value="item-1" className="border-none">
+                      <AccordionTrigger_Shadcn_ className="text-xs text-foreground-light font-normal gap-2 justify-start py-1 ">
+                        View syntax chart
+                      </AccordionTrigger_Shadcn_>
+                      <AccordionContent_Shadcn_>
+                        <CronSyntaxChart />
+                      </AccordionContent_Shadcn_>
+                    </AccordionItem_Shadcn_>
+                  </Accordion_Shadcn_>
                 </div>
               </FormControl_Shadcn_>
-              <FormMessage_Shadcn_ className="mt-1">
-                {scheduleString ? (
-                  <span className="text-foreground-light">
-                    The cron will be run {/* lowercase the first letter */}
-                    {scheduleString
-                      .split(' ')
-                      .map((s, i) => (i === 0 ? s.toLocaleLowerCase() : s))
-                      .join(' ')}
-                    .
-                  </span>
-                ) : (
-                  !fieldState.error && <span className="h-5" />
-                )}
+
+              <FormMessage_Shadcn_ className="mt-4">
+                <div className="mt-2 bg-surface-100 p-4 rounded grid gap-4 border">
+                  <h4 className="text-sm text-foreground">Schedule</h4>
+                  {scheduleString ? (
+                    <span
+                      className={cn(
+                        'text-xl font-mono',
+                        isGeneratingCron
+                          ? 'animate-pulse text-foreground-lighter'
+                          : 'text-foreground'
+                      )}
+                    >
+                      {isGeneratingCron ? <CronSyntaxLoader /> : presetValue || '* * * * * *'}
+                    </span>
+                  ) : (
+                    <span className="text-xl font-mono text-foreground-lighter">
+                      {isGeneratingCron ? <CronSyntaxLoader /> : presetValue || '* * * * * *'}
+                    </span>
+                  )}
+                  {scheduleString ? (
+                    <span className="text-foreground-light flex items-center gap-2">
+                      The cron will be run {/* lowercase the first letter */}
```

Comment:
this is an extension - but im thinking it might also be helpful to show when the job will run next - wdyt?

e.g if we selected every 5 minutes, and its currently 11.30pm, we can show "Job will run next at 11.35pm"

Reply:
e.g for the preset "Every hour, at 30 minutes" not sure if its just me but im still not sure how the job will run
clicking it shows the description "The cron will be run at 30 minutes past the hour." under the schedule which makes me wonder - "it's now 11.43pm, hmm does that mean it's gonna run at 12.30am?"

Reply:
Yes I think that's when it's set to run. Agreed on this one. I'll replace with an example that's less ambiguous, since docs / ai will prob do the heaviest lifting for people here.